Assets/Mirror/Runtime/NetworkManager.cs

```diff
@@ -756,7 +756,7 @@ public virtual void OnClientNotReady(NetworkConnection conn)
         public virtual void OnClientSceneChanged(NetworkConnection conn)
         {
             // always become ready.
-            ClientScene.Ready(conn);
+            //ClientScene.Ready(conn); to fixing "A connection has already been set as ready. There can only be one." error on respawning player
```

Comment:
why is this a comment?   just remove it.

Reply:
ok ... you can remove it :]